### DISCUSSION TITLE
[Suggestion] Add demonic/sinister offering to Prayer skill calc (300%)

### DISCUSSION THREAD

**Discussion by Corpselooter** [runelite/runelite#15231] [Q&A] [answered]
As title says, Would be nice if this could be added. Also works for the sacred bone burner in forthos. 

Thanks! would be great

**Comment by vmgi** [comment]
https://github.com/runelite/runelite/pull/15232

**Comment by Nightfirecat** [comment, marked as answer]
#15345 has added this
